support-frontend/app/services/RemindMeService.scala

```diff
@@ -0,0 +1,42 @@
+package services
+
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.lambda.AWSLambdaClientBuilder
+import com.amazonaws.services.lambda.model.InvokeRequest
+import com.gu.monitoring.SafeLogger
+import com.gu.support.config.{Stage, Stages}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class RemindMeService(stage: Stage) {
+
+  private val lambdaClient = AWSLambdaClientBuilder
+    .standard()
+    .withRegion(Regions.EU_WEST_1.getName)
+    .withCredentials(aws.CredentialsProvider)
+    .build()
+
+  private val functionName = stage match {
+    case Stages.PROD => "contributions-reminders-lambda-PROD"
+    case _ => "contributions-reminders-lambda-CODE"
+  }
+
+  def apply(email: String)(implicit ec: ExecutionContext): Future[Boolean] = {
+    val payload = s"""{"ReminderCreatedEvent": {"email": "$email"}}"""
```

Comment:
This might not work if for some random reason the email contains a `"`.

Reply:
I'm not sure I understand all the quotes here

Reply:
The triple quote opens a multiline string that allows the string contain literal " without escaping them.

I have changed it to first create a json object and then stringify it to send to the lambda.